cvat/apps/engine/media_extractors.py

```diff
@@ -29,6 +32,18 @@ def get_mime(name):
 
     return 'unknown'
 
+def get_pcd_properties(file):
+        kv = {}
+        for line_no, line in enumerate(file):
```

Comment:
why do you need line_no? I will propose to move parsing of PCD files into a separate py module.

Reply:
We have included all the parsing of PCD/BIN files in the class ValidateDimension.